chia/wallet/wallet_state_manager.py

```diff
@@ -2055,3 +2078,13 @@ async def get_or_create_vc_wallet(self) -> VCWallet:
             vc_wallet = await VCWallet.create_new_vc_wallet(self, self.main_wallet)
 
         return vc_wallet
+
+    async def sign_transaction(self, coin_spends: List[CoinSpend]) -> SpendBundle:
```

Comment:
why not keep this in the wallet?

Reply:
Because it has nothing to do with the standard wallet, not really.  Every wallet uses it, and it's just generic "find the right private key and sign this spend".